lib/rules/indent.js

```diff
@@ -168,7 +168,7 @@ module.exports = {
                 let rangeToFix = [];
 
                 if (needed > gotten) {
-                    const spaces = "" + new Array(needed - gotten + 1).join(indentChar);  // replace with repeat in future
+                    const spaces = `${new Array(needed - gotten + 1).join(indentChar)}`;  // replace with repeat in future
```

Comment:
I think this template is unnecessary since `Array.prototype.join` returns a string.
And we can use `indentChar.repeat(needed - gotten)` now 😉

http://node.green/#String-prototype-methods